config.libsonnet

```diff
@@ -54,6 +56,10 @@
     // servers under some non-root path.
     grafanaPrefix: '',
 
+    //Opt-in to multiCluster dashboards by overriding this and the clusterLabel
+    showMultiCluster: false,
+    clusterLabel: 'clusterCHANGEME',
```

Comment:
I think defaulting to `cluster` is fine.

Reply:
The thought for the CHANGEME was in case the user already had a cluster label, named 'cluster', then all the cluster!="" (when showMultiCluster is off and templates hidden) queries would bring back data for all the clusters. A corner case for sure.

Reply:
Perhaps in that case it would be better to have a `clusterSelector` metavar that is empty when `showMultiCluster` is off, and then sub that into all the selectors?

Reply:
Thought about that too. But preferred the readability of %(clusterLabel)s="$cluster" right in the dashboard.
Could go either way.

Do you have a preference?